.github/workflows/action.yml

```diff
@@ -28,7 +31,7 @@ jobs:
       - run: npm run yamllint
       - run: npm run jslint
       - name: Check if build left artifacts
-        run: git diff --exit-code
+        run: git diff --exit-code --ignore-space-at-eol
```

Comment:
I get strange behaviour without this, in that the same number of additions and subtractions is present in the diff, but it exits with a code other than 0. If somebody wants me to dig deeper, I could, but I'm not sure how relevant this is in our context (the goal here is making sure that whatever is pushed is what is present locally when running `npm run build-dist`).

Reply:
I experienced the same on my current PR. The git version being used on github side maybe changed.